prow/plugins.yaml

```diff
@@ -4,32 +4,25 @@
 ---
 triggers:
 - repos:
+  - GoogleCloudPlatform/k8s-multicluster-ingress
+  - google/cadvisor
   - kubernetes
   - kubernetes-incubator
   - kubernetes-security
   - kubernetes-sigs
-  - google/cadvisor
-  - tensorflow/k8s
-  - tensorflow/minigo
```

Comment:
is tf/minigo also pointing to somewhere in kubeflow?

Reply:
yeah, tensorflow/k8s moved to https://github.com/kubeflow/tf-operator but tensorflow/minigo is still at tensorflow/minigo ...?

Reply:
Ack

Reply:
You can ask Josh Hoak about tensorflow/minigo . tensorflow/k8s is now part of Kubeflow so thats fine to remove.